test/augmentation/test_augmentation.py

```diff
@@ -3483,6 +3485,50 @@ def test_module(self, device, dtype):
         pass
 
 
+class TestRandomChannelDropout(BaseTester):
+    def test_smoke(self, device, dtype):
+        torch.manual_seed(0)
+        img1 = torch.arange(1 * 3 * 2 * 2, device=device, dtype=dtype).view(1, 3, 2, 2)
+
+        out_expected1 = torch.tensor(
+            [[[[0.0, 1.0], [2.0, 3.0]], [[0.0, 0.0], [0.0, 0.0]], [[8.0, 9.0], [10.0, 11.0]]]],
+            device=device,
+            dtype=dtype,
+        )
+
+        img2 = torch.arange(1 * 2 * 2 * 2, device=device, dtype=dtype).view(1, 2, 2, 2)
+        out_expected2 = torch.tensor([[[[0.0, 0.0], [0.0, 0.0]], [[4.0, 5.0], [6.0, 7.0]]]], device=device, dtype=dtype)
+
+        aug1 = RandomChannelDropout(channel_drop_range=(1, 2), p=1.0)
+        out1 = aug1(img1)
+
+        aug2 = RandomChannelDropout(channel_drop_range=(1, 1), p=1.0)
+        out2 = aug2(img2)
+
+        self.assert_close(out1, out_expected1)
+        self.assert_close(out2, out_expected2)
+
+    @pytest.mark.skip(reason="not implemented yet")
```

Comment:
i you don't implement maybe it's not worth to subclass from BaseTester. @johnnv1 we might rethink about different test cases

Reply:
assert_close does not work without BaseTester, so, for consistency, I assume it is ok (?)